scripts/cloudfoundry/bat-cdn/bat-cdn.rb

```diff
@@ -11,6 +11,8 @@
     "domain" => input[_env]["domain"].join(",")
 }
 
-cmd = "cf update-service bat-cdn-#{_env} -c '#{output.to_json}'"
+service=input[_env]["service"] || "bat-cdn-#{_env}"
```

Comment:
It would be cleaner to add a `service` entry for bat in the yaml and rename `prod` to `bat-prod` etc?

Reply:
yes, I can see that.